include/substrait-mlir/Dialect/Substrait/IR/SubstraitOps.td

```diff
@@ -394,6 +394,9 @@ def Substrait_UnionDistinct_Op : Substrait_RelOp<"union_distinct"> {
   );
 
   let results = (outs Substrait_Relation:$result);
+  let assemblyFormat = [{
+    $left `u_d` $right attr-dict `:` type($left) `u_d` type($right) `->` type($result)
```

Comment:
I am not a huge fan of the `u_d` "operator" here, TBH. I see how this is an evolution of the `x` in `CrossOp`, which makes sense. But currently the op name already produces the information "union" and "distinct" so the "operator" only needs to separate left from right input. `u` or `,` would be sufficient to do that. I also didn't immediate understand what `u_d` stands for.

TL;DR: For this PR, I suggest to use `u` or `,`. In some subsequent PR, we have to think about how to represent the various variants anyways, so the choice isn't hugely important.

Reply:
Okay makes sense. I'll go with "u".